src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { Redirect } from 'react-router';
+import { Form } from 'react-bootstrap';
+
+let SearchBar = ({ searchParams }) => {
+    const [searchParam, setsearchParam] = useState(searchParams);
+    const [redirectToSearchPage, setRedirectToSearchPage] = useState(false);
+
+    const handleSubmit = (e) => {
+        setRedirectToSearchPage(true);
+        e.preventDefault();
+    };
+    const updateSearchParam = (e) => {
+        setsearchParam(e.target.value);
+    };
+    if (redirectToSearchPage === true) {
+        return <Redirect to={`/search/?search=${searchParam}`} />;
+    }
+    return (
+        <Form className="d-flex" onSubmit={(e) => handleSubmit(e)}>
+            <Form.Control
+                type="text"
+                name="search"
+                placeholder="Search UBook"
+                onChange={(e) => updateSearchParam(e)}
+                value={searchParam ? searchParam : ''}
+            ></Form.Control>
```

Comment:
no need of adding the closing tag for Form.Control. 
```
<Form.Control
                type="text"
                name="search"
                placeholder="Search UBook"
                onChange={(e) => updateSearchParam(e)}
                value={searchParam ? searchParam : ''}
            />
```

Reply:
fixed